src/pages/accessibility/index.mdx

```diff
@@ -85,10 +85,10 @@ Applicable Standards/Guidelines: This report covers the degree of conformance fo
 | [1.4.3 Contrast (Minimum)](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast) (Level AA)                      | Supports           | All default user interface elements feature an appropriate contrast ratio. Users are responsible for ensuring the accessibility of any custom content they create.   |
 | [1.4.4 Resize text](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-scale) (Level AA)                                | Supports           | Text can be resized using native browser controls.                                                                                                                   |
 | [1.4.5 Images of Text](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-text-presentation) (Level AA)                 | Supports           | PrairieLearn does not contain images of text by default. Users are responsible for ensuring the accessibility of any images containing text they upload.             |
-| [1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow) (Level AA 2.1 only)                                               | Partially Supports | Some pages such as the instructor questions list and gradebook pages do not display properly at a resolution of 320x256.                                             |
+| [1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow) (Level AA 2.1 only)                                               | Supports           | Some pages such as the instructor questions list and gradebook pages feature data tables and thus require scrolling in two dimensions at small viewport sizes.       |
 | [1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast) (Level AA 2.1 only)                         | Supports           | User interface components and graphical objects have a contrast ratio of at least 3:1 against adjacent colors.                                                       |
 | [1.4.12 Text Spacing](https://www.w3.org/TR/WCAG21/#text-spacing) (Level AA 2.1 only)                                   | Supports           | Users can change text spacing without loss of content or functionality.                                                                                              |
-| [1.4.13 Content on Hover or Focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) (Level AA 2.1 only)         | Partially Supports | If additional content was triggered via focus, it cannot be dismissed without removing focus.                                                                        |
+| [1.4.13 Content on Hover or Focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) (Level AA 2.1 only)         | Partially Supports | Custom tooltips cannot be dismissed without removing pointer hover or keyboard focus, and their content will disappear if the pointer is moved over them.            |
```

Comment:
This will be resolved if/when this PR lands: https://github.com/twbs/bootstrap/pull/35151